homeassistant/components/tplink/entity.py

```diff
@@ -231,6 +231,7 @@ def __init__(
         """Initialize the entity."""
         self.entity_description = description
         super().__init__(device, coordinator, parent=parent, feature=feature)
+        self._async_call_update_attrs()
```

Comment:
Should this be in async_added_to_hass instead?

Reply:
Done, it was already added to `async_added_to_hass` in the binary_sensor PR, but I forgot to update this one accordingly.